### PR TITLE
archive_extract: cleanup, add tests

### PR DESCRIPTION
**Pull request by jmairboeck**
This is a test for #9394. If this works and is accepted, the same pattern could be used for other perl packages.

### CODE REVIEW

**Comment by jmairboeck** [comment]
I added some cleanup in a second commit to remove some architecture specific files. Something similar should probably be done for all "any" arch perl packages.

**Comment by Begasus** [comment]
LGTM, will try  this with the other ones and see if we can get frozen-bubble up and running on 32bit :)

File: dev-perl/archive_extract/archive_extract-0.88.recipe [hunk 1]
```diff
@@ -38,5 +39,10 @@ BUILD()
...
 INSTALL()
 {
-	make install DESTDIR="${DESTDIR}"
+	make install
+
+	# remove architecture-specific files
+	cd $prefix
+	rm -r $(perl '-V:archlib|vendorarch' | cut -d\' -f2 | cut -d/ -f5-)
+		# cut extracts the quoted string and strips the prefix (which is perl's and not ours)
 }
```
Comment (Begasus) [inline dev-perl/archive_extract/archive_extract-0.88.recipe:48]: This is a problem with compress_bzip2

Comment (jmairboeck) [reply dev-perl/archive_extract/archive_extract-0.88.recipe:48]: This is only meant for "any" arch recipes, not for architecture specific ones.

**Comment by jmairboeck** [comment]
I reverted the secondaryArch stuff now that this is finally settled and just kept the cleanup and test changes in.

**Pull request merged**